### DISCUSSION TITLE
How do i cancel/delete queued plots without restarting Chia and loosing active plots ? is there a CLI command ?

### DISCUSSION THREAD

**Discussion by brahmasatya** [Chia-Network/chia-blockchain#3169] [General]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

- OS: [e.g. Linux]
- OS Version/Flavor: [e.g. CentOS 7.2]
- CPU: [e.g. Intel Xeon 8175M]

**Additional context**

**Comment by mumtazali2** [comment]
Yes, this is a very important issue. Waiting for any resolution on this!

**Comment by lvcivs** [comment]
If you start plots from the CLI with `chia plots create` you don't have this problem, it will spawn a process independent from the GUI.

**Comment by sargonas** [comment]
Tagging this issue for migration to a discussion. Apologies in advance, but you will be best served by moving this over there and out of the issue tracker, where the community can better assist you. (Ideally the issue tracker should be used to focus on bugs and development problems that need to be addressed, with general user support handled in Discussions and Keybase)

^MigrateToDiscussion
